tests/errors_test.py

```diff
@@ -2,35 +2,37 @@
 from tests.printer_spy import PrinterSpy
 
 
-def test_errors_can_print_invalid_input_error():
+def test_errors_can_print_input_not_numeric_error_message():
     spy = PrinterSpy()
     error = Errors(spy)
 
-    error.invalid_input_error()
+    error.input_not_numeric_error_message()
 
-    expected = "Sorry, that input is invalid. Please enter numbers 1 - 9:\n"
+    expected = "\nI didn't understand that ... trying entering a number: 1 - 9:\n"
 
     assert spy.printed() == expected
 
 
-def test_errors_can_print_index_out_of_range_error():
+def test_errors_can_print_invalid_board_index_error_message():
     spy = PrinterSpy()
     error = Errors(spy)
 
-    error.index_out_of_range_error()
+    error.invalid_board_index_error_message()
 
-    expected = "Sorry, that input is out of range. Please enter 1 - 9:\n"
+    expected = "\nSorry, friend ... that number isn't one of the squares. Please enter 1 - 9:\n"
 
     assert spy.printed() == expected
```

Comment:
You can strip the newlines off a string in python with `mystr.strip()`. You could change this line to
```
assert spy.printed().strip() == expected.strip()
```
That way you wouldn't have to worry about including any newlines in your `expected` string.